scrapers/scrape_common.py

```diff
@@ -297,6 +367,8 @@ def parse_xls(book, header_row=1, sheet_index=0, sheet_name=None, skip_rows=1, c
                 entry[h] = xlrd.xldate.xldate_as_datetime(value, book.datemode)
             elif cell_type == xlrd.XL_CELL_EMPTY:
                 entry[h] = None
+            elif support_float and represents_float(value):
```

Comment:
why do we need this flag? Is there situation, where float should nlt be supported?

Reply:
Each `int` value is also a `float` value, that's why I added the flag and `represents_float` function.
I hope to have improved this in ffb3ca7: the `represents_int` is extended to verify that the value is actually an integer and if that's not the case, it's being checked for `float`.
Do you think this makes more sense?